### ISSUE TITLE
TypeError: panel.hidePopup is not a function

### ISSUE THREAD

**Issue by pdehaan** [biancadanforth/tracking-protection-shield-study#68]
https://github.com/biancadanforth/tracking-protection-shield-study/blob/ce8f96c75659b2f46a5394689d660c2c8fc0b39b/addon/lib/Feature.jsm#L656

```
TypeError: panel.hidePopup is not a function      Feature.jsm:656:5
```

![browser_console](https://user-images.githubusercontent.com/557895/35541084-dfad8ff0-050d-11e8-98d9-21d338d7c544.png)

**Comment by biancadanforth** [comment]
This was happening to me as well, since when you change a tab, the pageAction button in the URL bar gets removed (along with its child element, the <panel>). In this PR, I added a check to see if the <panel> exists in the browser.xul document before trying to remove it.

@pdehaan , I claim this was fixed in PR #67 (here is the [exact line](https://github.com/biancadanforth/tracking-protection-shield-study/commit/65d7239760d9c4e7fb658c54e9df85fcc45139f0#diff-82517ac932350429ca2aecf5c948c1f7R657)), which was merged less than 24 hours ago. Is there any chance you did not fetch the latest changes before trying this?

Let me know if you are still receiving the error. I'm not.
